src/util/graphql.js

```diff
@@ -1,6 +1,9 @@
+import fetch from 'isomorphic-fetch'
```

Comment:
Oh are we still specifically importing fetch? I think I used it 'raw' in a reactions PR... which might be a gap on older browsers (has fetch crossed the line into availability??)